src/org/sparkboard/server/slack/screens.clj

```diff
@@ -162,6 +174,22 @@
        :options [{:value "collect-in-thread"
                   :text [:md "Collect responses in a thread"]}]}]]]))
 
+(defn state-test-modal [context payload state]
```

Comment:
I guess this is temporary?

Reply:
yeah, this is just a little test modal to see how the local-state stuff behaves.